tests/UI/campaigns/utils/globals.js

```diff
@@ -21,7 +21,7 @@ global.INSTALL = {
   DB_NAME: process.env.DB_NAME || 'prestashopdb',
   DB_USER: process.env.DB_USER || 'root',
   DB_PASSWD: process.env.DB_PASSWD || '',
-  SHOPNAME: process.env.SHOPNAME || 'Prestashop',
+  SHOPNAME: process.env.SHOPNAME || 'PrestaShop',
   PS_VERSION: process.env.PS_VERSION || '1.7.6.0',
```

Comment:
We might update this variable, what do you think ? Like `1.7.6.8` ?

Reply:
Okey, I will update it :+1:

Reply:
The variable is used only in upgradeShop test, we will update it when we fix the test :+1: